launcher/ui/themes/ThemeManager.cpp

```diff
@@ -0,0 +1,154 @@
+// SPDX-License-Identifier: GPL-3.0-only
+/*
+ *  Prism Launcher
+ *  Copyright (C) 2022 Tayou <tayou@gmx.net>
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, version 3.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#include "ThemeManager.h"
+
+#include <versionhelpers.h>

```

Comment:
You need to remove this include as this is a windows only file and you've already included it after:

https://github.com/PrismLauncher/PrismLauncher/blob/55881674be900625ab57e982a72262c820719349/launcher/ui/themes/ThemeManager.cpp#L32-L35

Reply:
ahh, I messed it up when rebasing and moving my local branches around... 😭

Reply:
no worries, just pointing this out ^^